Makefile

```
@@ -370,6 +370,13 @@ verify-boilerplate: ## Verify boilerplate text exists in each file
 verify-container-images: ## Verify container images
 	TRACE=$(TRACE) ./hack/verify-container-images.sh
 
+.PHONY: verify-flavors
+verify-flavors: $(FLAVOR_DIR) generate-flavors
+	@if !(git diff --quiet HEAD -- templates); then \
+		git diff; \
```

```suggestion
	@if !(git diff --quiet HEAD -- $(FLAVOR_DIR)); then \
		git diff $(FLAVOR_DIR); \
```

Let's use the parameter for both. paths :-)

---

Done